chain/index/chain-indexing-overview-for-rpc-providers.MD

```diff
@@ -0,0 +1,258 @@
+# ChainIndexer Documentation for RPC Providers
+
+# Table of Contents
+
+- [ChainIndexer Documentation for RPC Providers](#chainindexer-documentation-for-rpc-providers)
+  - [Introduction](#introduction)
+  - [ChainIndexer Config](#chainindexer-config)
+    - [Enablement](#enablement)
+    - [Garbage Collection](#garbage-collection)
+      - [Recommendations](#recommendations)
+    - [Removed Options](#removed-options)
+  - [Migration Guide](#migration-guide)
+    - [Backfilling](#backfilling)
+      - [The `ChainValidateIndex` JSON RPC API](#the-chainvalidateindex-json-rpc-api)
+      - [`lotus-shed chainindex validate-backfill` CLI tool](#lotus-shed-chainindex-validate-backfill-cli-tool)
+        - [Usage](#usage)
+        - [Parameters](#parameters)
+        - [Error conditions](#error-conditions)
+        - [Logging](#logging)
+        - [Example usage](#example-usage)
+  - [Rollback](#rollback)
+  - [Need more help?](#need-more-help)
+
+## Introduction
+
+This document is aimed at RPC providers and node operators who serve RPC requests and aims to walk through the configuration changes, migration flow and operations/maintenance work needed to enable, backfill and maintain the `ChainIndexer`.
+
+**Note: If you are a Storage Provider or node operator who does not serve RPC requests, you can skip this document as the `ChainIndexer` is already disabled by default**. 
+ 
+
+## ChainIndexer Config
+### Enablement
+
+The following must be enabled on an RPC node before starting as they are disabled by default:
+
+```toml
+[ChainIndexer]
+# Enable the ChainIndexer. 
+  EnableIndexer = true 
+
+[Fevm]
+# Enable the ETH RPC APIs.
+  EnableEthRPC = true
+
+[Events]
+# Enable the Actor Events APIs.
+  EnableActorEventsAPI = true
+```
+
+You can learn more about these configuration options and other configuration options available for the `ChainIndexer` [here](https://github.com/filecoin-project/lotus/blob/master/documentation/en/default-lotus-config.toml).
+
+
+### Garbage Collection
+
+The `ChainIndexer` includes a garbage collection (GC) mechanism to manage the amount of historical data retained. By default, GC is disabled to preserve all indexed data.
+
+To configure GC, use the `GCRetentionEpochs` parameter in the `ChainIndexer` section of your config.
+
+The ChainIndexer periodically runs GC if `GCRetentionEpochs` is > 0 and removes indexed data for epochs older than `(current_head_height - GCRetentionEpochs)`.
```

Comment:
```suggestion
The ChainIndexer [periodically runs](https://github.com/filecoin-project/lotus/blob/master/chain/index/gc.go#L15) GC if `GCRetentionEpochs` is > 0 and removes indexed data for epochs older than `(current_head_height - GCRetentionEpochs)`.
```

Linking to the source of truth for where someone can learn how frequently this runs.